### PR TITLE
Remove the in construction section, and the statement is not in final form

### PR DESCRIPTION
**Pull request by Carreau**
Remove the `This is the current text of the statement. It is not yet final, and your feedback is welcome.
` part will add a `discussion` section below in separate RP that invite feedback.

### CODE REVIEW

**Comment by takluyver** [comment]
:+1: While we're at it, can we consider changing the rather luminous yellow background to something a bit softer?

Do you want to add the discussion section in this PR?

**Comment by Carreau** [comment]
> While we're at it, can we consider changing the rather luminous yellow background to something a bit softer?

Updated to be greenish + 2 sections at the end, respective purple and Blue:

```
## Discuss
While the above statement is still is close to final form it is still open for discussion. Feel free to open an issue on or GitHub repository or browse the open and closed ones.

If you think some information are missing, or inaccurate, please let us know as well.


## Update your project to be Python 3 only
While most of the project that have signed up above are migrating to Python 3 only code base, we are gathering information about the difficulties that can be encountered both by developers and users during this process.

We are thus collecting a list of things to be aware of for the transition to be as seem less as possible. For example : as a developer, how to make sure that package manager like pip do not upgrade you your libraries to incompatible versions on Python 2. Check our repository for more informations.
```

File: _sections/70-discuss.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+title: discuss
+fa-icon: comment
+bg: '#a066f3'
+color: 'white'
+style: center
+---
+
+# Discuss
+
+While the above statement is still is close to final form it is still open for
```
Comment (takluyver) [inline _sections/70-discuss.md]: Take out the first 'is still' here, and add a comma after 'form'.

File: _sections/70-discuss.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+discussion. Feel free to [open an
+issue](https://github.com/python3statement/python3statement.github.io/issues)
+on or GitHub repository or browse the open and closed ones.
```
Comment (takluyver) [inline _sections/70-discuss.md]: *our* GitHub repository

File: _sections/70-discuss.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+---
+
+# Discuss
+
+While the above statement is still is close to final form it is still open for
+discussion. Feel free to [open an
+issue](https://github.com/python3statement/python3statement.github.io/issues)
+on or GitHub repository or browse the open and closed ones. 
+
+If you think some information are missing, or inaccurate, please let us know as
```
Comment (takluyver) [inline _sections/70-discuss.md]: information *is*, and remove the comma after missing.

File: _sections/80-update-my-project.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
+---
+title: Update
+fa-icon: arrow-circle-up
+bg: '#49a7e9'
+color: 'white'
+style: center
+---
+
+# Update your project to be Python 3 only
```
Comment (takluyver) [inline _sections/80-update-my-project.md]: I'd call this section something like 'Practicalities' - but it may be easiest to leave it for a future PR.

**Comment by Carreau** [comment]
Done, done, done, and done.

**Comment by takluyver** [comment]
OK, let's see what it looks like :-)

**Pull request merged**